### ISSUE TITLE
When synchronizing the translation units, a translation is lost if it has at the beginning an interpolation

### ISSUE THREAD

**Issue by olalinv** [rvanbekkum/vsc-xliff-sync#67]
I´ve got this translation unit in messages.xlf:
```
<trans-unit id="CreateModal.MessageOK" datatype="html">
  <source>Ticket <x id="INTERPOLATION" equiv-text="{{createResponseOK.code}}"/> created successfully</source>
</trans-unit>
```
And this is the tranlation in messages.eu-ES.xlf:
```
<trans-unit id="CreateModal.MessageOK" datatype="html">
  <source>Ticket <x id="INTERPOLATION" equiv-text="{{createResponseOK.code}}"/> created successfully</source>
  <target state="translated"><x id="INTERPOLATION" equiv-text="{{createResponseOK.code}}"/> eskaera behar bezala sortu da</target>
</trans-unit>
```
By synchronizing the translation units again the translation becomes:
```
<trans-unit id="CreateModal.MessageOK" datatype="html">
  <source>Ticket <x id="INTERPOLATION" equiv-text="{{createResponseOK.code}}"/> created successfully</source>
  <target state="translated">Ticket </target>
</trans-unit>
```
If I put a space (or any word) just before the interpolation then this doesn't happen:
`<target state="translated">&nbsp;<x id="INTERPOLATION" equiv-text="{{createResponseOK.code}}"/> eskaera behar bezala sortu da</target>`

**Comment by rvanbekkum** [comment]
Hi @olalinv,
Sorry for the late reply. I have looked into your issue, but I can't seem to reproduce it yet.
For me the translation does not disappear, however, what I do see is that the target node is being marked as a missing-translation. I can see why that happens and will look into fixing that.

However, that is not the bug you reported, so could you provide me with a sample project which reproduces this? (Maybe it is caused by specific settings.)

**Comment by rvanbekkum** [comment]
Back at it again. Found another example of reproducing this, which is when copyFromSource is used. The issue is that currently nodes nested in the target are not supported. I will look into this though.

**Comment by olalinv** [comment]
Hi @ rvanbekkum,

This is my settings file:

`{
  "xliffSync.baseFile": "messages.xlf",
  "xliffSync.addNeedsWorkTranslationNote": false,
  "xliffSync.copyFromSourceForLanguages": [
    "ca-ES",
    "de-DE",
    "en-US",
    "eu-ES",
    "es-ES",
    "fr-FR",
    "gl-ES",
    "it-IT",
    "pt-PT"
  ],
  "xliffSync.detectSourceTextChanges": false
}`

I will try to prepare a sample project as soon as i can for further testing, if you need help please let me know.

Thanks

**Comment by rvanbekkum** [comment]
A change for this will be shipped in the next release of the extension!

**Comment by rvanbekkum** [comment]
Shipped in v1.0.0.

**Comment by olalinv** [comment]
I've tried it and it works perfectly. Thanks a lot.
